web_src/less/_review.less

```diff
@@ -16,6 +16,13 @@
   }
 }
 
+.diff-file-box .lines-code .code-inner.has-escaped::before {
+  visibility: visible;
+  content: '⚠️';
```

Comment:
In the diff view, these throw off alignment of code. For example, with the file

```
a normal line
```

If that line is changed to

```
a line with badbidi
```

The diff will display as

```diff
-a normal line
+⚠️ a line with badbidi
```

And the icon stays regardless of escaped status. However this works fine in the regular file view.

Reply:
It should probably be added on the left side of the line numbers, probably with some negative margins so we won't have to enlarge the column significantly.

Reply:
What browser are you using?

Reply:
It looks like something this was relying on has changed and it no longer works!

Reply:
@gwymor you are more than welcome to try to find out where this can go.

Let's look at html:

```html
<tr class="add-code nl-5 ol-5" data-line-type="add">
	<td class="lines-num lines-num-old" data-line-num=""><span rel=""></span></td>
	<td class="lines-num lines-num-new" data-line-num="105"><span rel="diff-8ec9a00bfd09b3190ac6b22251dbb1aa95a0579dR105"></span></td>
	<td class="lines-type-marker"><span class="mono" data-type-marker="+"></span></td>
	<td class="chroma lines-code"><code class="code-inner has-escaped" title="This line has hidden unicode characters">if access_level != "user<span class="escaped-code-point" data-escaped="[U+202E]"><span class="char">‮</span></span> <span class="escaped-code-point" data-escaped="[U+2066]"><span class="char">⁦</span></span>// Check if admin<span class="escaped-code-point" data-escaped="[U+2069]"><span class="char">⁩</span></span> <span class="escaped-code-point" data-escaped="[U+2066]"><span class="char">⁦</span></span>" {</code></td>
</tr>
```

* We can't put the :warning: before `<td class="lines-num lines-num-old">` because that's use for the line number
* We can't put the :warning: before `<td class="lines-num lines-num-new">` because that's use for the line number
* I tried putting it before the `<td class="lines-type-marker">` but that ruins the line-marker
* I tried adding adjacent to the `<span class="mono" data-type-marker="+">`  either before or after but that ruins the alignment there.

Where do you propose putting this?

Reply:
I've added another column.